cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -204,8 +204,14 @@ features:
     TopPanel:
       SchemaModal:
         defaultErrorMessage: Error generating schema.
+        invalidFieldNameMessage: Invalid field name "{fieldName}"
+        invalidFieldNameRemedies1: "Your field name contains an invalid character: no space or special character other than - or _ are allowed"
+        invalidFieldNameRemedies2: Try to apply directive “cleanse-column-names”
       PipelineModal:
         defaultErrorMessage: Error adding to pipeline
+        invalidFieldNameMessage: Invalid field name "{fieldName}"
+        invalidFieldNameRemedies1: "Your field name contains an invalid character: no space or special character other than - or _ are allowed"
+        invalidFieldNameRemedies2: Try to apply directive “cleanse-column-names”
```

Comment:
Hhmm this is anyways a temporary solution. Will be removed once the errors come from backend.